board/batocera/fsoverlay/etc/init.d/S08connman

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+case "$1" in
+	start)
+		printf "Starting connman ... "
+		start-stop-daemon -S -q -m -b -p /var/run/connmand.pid --exec /usr/sbin/connmand -- -n
```

Comment:
```
start-stop-daemon -S -q -m -p /var/run/connmand.pid --exec /usr/sbin/connmand
```

and remove the sleep
Do not send connmand to background per start-stop-daemon `-b` command and use the natural behaviour of connmand (because -n also does not allow the send connmand to background)

So with this codeline start-stop-daemon can be used to analyze error codes and connmand goes to background per automatic. It works for me.

Reply:
Just saw that you can't catch PID with this one
```
                /usr/sbin/connmand
                pgrep -f "/usr/sbin/connmand" > /var/run/connmand.pid
```